<?xml version="1.0"?>
<!--
Launch the path follower with the stanley controller for a car-like (Ackermann) robot.
-->
<launch>
    <group ns="path_follower">
        <group ns="controller">
            <!-- parameters, that are dependent of the controller or the robot model -->
            <group ns="ackermann_stanley">
                <!-- simulation -->
                <param name="k_forward" value="1.0" />
                <param name="k_backward" value="3.0" />
                <param name="factor_steering_angle" value="1.0" />

                <!-- robot -->
                <!-- param name="k_forward" value="0.7" />
                <param name="k_backward" value="0.7" />
                <param name="factor_steering_angle" value="0.5" / -->
                <param name="vehicle_length" value="0.34" />
            </group>
        </group>
    </group>  
</launch>
